src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days >= 1 && days < 3) {
+    total = days * 40;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
Перевірку на >=3 можна прибрати. Тому що якщо не пройде перша, то днів вже точно буде не менше трьох.
А якщо буде менше, то до цього else if'а код не дійде

Reply:
Исправила)